src/components/Login.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+    <main>
+        <div class="container">
+            <main class="form-signin text-center">
+                <form @submit.prevent="handleSubmit">
+                    <a href="/">
+                        <img class="mb-4" src="/src/assets/pizza.png" alt="" width="72" height="57">
+                    </a>
+                    <h1 class="h3 mb-3 fw-normal">Welcome to Pizza Hub</h1>
+                    <h2 class="h3 mb-3 fw-normal">Please Log in</h2>
+
+                    <div v-if="showSuccess" class="alert alert-success d-flex align-items-center" role="alert">
+                        <BootstrapIcon icon="check-circle-fill" /> 
+                    <div>
+                        Login successful.
+                    </div>
+                    </div>
+
+                    <div v-if="showFailed" class="alert alert-danger d-flex align-items-center alert-dismissible fade show" role="alert">
+                        <BootstrapIcon icon="exclamation-triangle-fill" /> &nbsp;
+                        <div>
+                            Error occurred. Try again.
+                            <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+                        </div>
+                    </div>
+                    
+                    <div class="form-floating">
+                        <input 
+                            required
+                            type="email" 
+                            class="form-control" 
+                            id="floatingInput" 
+                            placeholder="name@example.com"
+                            v-model="email">
+                        <label for="floatingInput">Email Address</label>
+                    </div>
+                    <div class="form-floating">
+                        <input 
+                            required
+                            type="password" 
+                            class="form-control" 
+                            id="floatingPassword" 
+                            placeholder="Password"
+                            v-model="password">
+                        <label for="floatingPassword">Password</label>
+                    </div>
+                
+                    <div class="checkbox mb-3">
+                        <label>
+                            <input type="checkbox" value="remember-me"> Remember me
```

Comment:
Do you need this as you are not handling cookies yourself?

Reply:
I actually do not need it, as handling cookies right is not one of the functional items.